metrics/web/test/common/presentation/text_placeholder/widgets/text_placeholder_test.dart

```diff
@@ -21,58 +20,58 @@ void main() {
     );
 
     testWidgets(
-      "displays the given text in the text placeholder",
+      "displays the given text",
       (WidgetTester tester) async {
         const text = 'text';
 
         await tester.pumpWidget(
-          const _MetricsTextPlaceholderTestbed(text: text),
+          const _TextPlaceholderTestbed(text: text),
         );
 
-        expect(find.widgetWithText(TextPlaceholder, text), findsOneWidget);
+        expect(find.text(text), findsOneWidget);
       },
     );
 
     testWidgets(
-      "applies the text style from the metrics inactive widget theme to the text placeholder",
+      "applies the text style from the text placeholder theme to the text",
       (WidgetTester tester) async {
         const textStyle = TextStyle(
           color: Colors.red,
           fontWeight: FontWeight.bold,
         );
 
         const theme = MetricsThemeData(
-          inactiveWidgetTheme: MetricWidgetThemeData(textStyle: textStyle),
+          textPlaceholderTheme: TextPlaceholderThemeData(textStyle: textStyle),
         );
 
         await tester.pumpWidget(
-          const _MetricsTextPlaceholderTestbed(theme: theme),
+          const _TextPlaceholderTestbed(theme: theme),
         );
 
-        final textPlaceholder = tester.widget<TextPlaceholder>(
-          find.byType(TextPlaceholder),
+        final text = tester.widget<Text>(
+          find.byType(Text),
```

Comment:
Maybe it would be more reliable to find the placeholder text by the given text, not type?